src/app/components/form/DualCurrencyField/index.tsx

```diff
@@ -85,8 +91,13 @@ export default function DualCurrencyField({
           {label}
         </label>
         {(min || max) && (
-          <span className="text-xs text-gray-700 dark:text-neutral-400">
-            <RangeLabel min={min} max={max} /> sats
+          <span
+            className={classNames(
+              "text-xs text-gray-700 dark:text-neutral-400",
+              (!!min || !!max) && !!rangeExceeded && "text-red-500"
```

Comment:
Why is `(!!min || !!max)` check needed here? it is already checked in the enclosing scope (line 93)

Reply:
You shouldn't need to do `!!` on a boolean? values

Reply:
The reason why I had `(!!min || !!max)` here is, in Keysend component I have `const rangeExceeded = +amountSat < amountMin;` - `amountMin` is `1` and if for some reason `amountSat` would be `undefined` I would end up with `+undefined < 1` which is `false`. This cased a small amount of uncertainty and I wanted to make sure this rangeExceeded is only being applied when there actually is a min or max value. But probably it is not needed.